src/PKSim.Presentation/Presenters/Compounds/MolWeightHalogensPresenter.cs

```diff
@@ -36,33 +33,14 @@ public override void InitializeWith(ICommandCollector commandCollector)
          _parameterEditPresenter.InitializeWith(commandCollector);
       }
 
-      public override void ReleaseFrom(IEventPublisher eventPublisher)
+      public void EditHalogens(IEnumerable<IParameter> halogenParameters)
       {
-         base.ReleaseFrom(eventPublisher);
```

Comment:
All of that is done automatically when adding the presenter as subpresenter... Very old code here!